Release notes for scroll timeline (#12410)
mdn/content: files/en-us/mozilla/firefox/releases/97/index.md

- The CSS units `cap` and `ic` are now supported for use with {{cssxref("&lt;length&gt;")}} and {{cssxref("&lt;length-percentage&gt;")}} data types.
  For more information see {{bug(1702924)}} and {{bug(1531223)}}.

> - The {{cssxref('@scroll-timeline')}} at-rule and {{cssxref('animation-timeline')}} property are now supported. This allows developers to define an [`AnimationTimeline`](/en-US/docs/Web/API/AnimationTimeline) whose time values are determined by progression of scrolling within a scroll container and not by minutes or seconds. Once specified, a scroll timeline is associated with a [CSS Animation](/en-US/docs/Web/CSS/CSS_Animations) by using the `animation-timeline` property.

Reviewer: ```suggestion
- The {{cssxref('@scroll-timeline')}} at-rule and {{cssxref('animation-timeline')}} property are now supported. This allows you to define an [`AnimationTimeline`](/en-US/docs/Web/API/AnimationTimeline), of which time values are determined by scrolling progress within a scroll container and not by minutes or seconds. Once specified, a scroll timeline is associated with a [CSS Animation](/en-US/docs/Web/CSS/CSS_Animations) by using the `animation-timeline` property.
```

Some light style/concision changes. Optional, if you don't like 'em.
